### PR TITLE
Adding JS specific syntax to cheatsheet

### PR DESCRIPTION
**Pull request by gnomeontherun**
This is a first pass at getting the JS syntax into the cheatsheet. I am not confident about some of the syntax and would like a review.

### CODE REVIEW

**Comment by naomiblack** [comment]
@IgorMinar - any reason not to push this live and edit later, assuming it renders ok?

**Comment by IgorMinar** [comment]
push. I'll review it online.

File: modules/angular2/docs/cheatsheet/bootstrapping.md [hunk 1]
```diff
@@ -2,12 +2,15 @@
 Bootstrapping
 @cheatsheetIndex 0
 @description
-{@target js ts}`import {bootstrap} from 'angular2/platform/browser';`{@endtarget}
+{@target ts}`import {bootstrap} from 'angular2/angular2';`{@endtarget}
+{@target js}Available on the `ng.platform.browser` namespace.{@endtarget}
```
Comment (IgorMinar) [inline modules/angular2/docs/cheatsheet/bootstrapping.md]: "on" -> "from"

File: modules/angular2/docs/cheatsheet/directive-and-component-decorators.md [hunk 2]
```diff
@@ -2,62 +2,79 @@
...
 Declares an input property that we can update via property binding (e.g.
 `<my-cmp [my-property]="someExpression">`).
 
 
 @cheatsheetItem
-syntax:
+syntax(ts dart):
 `@Output() myEvent = new EventEmitter();`|`@Output()`
+syntax(js):
+`myEvent = new EventEmitter(); ng.core.Output(myEvent, myComponent);`|`ng.core.Output(`|`);`
```
Comment (IgorMinar) [inline modules/angular2/docs/cheatsheet/directive-and-component-decorators.md]: `new EventEmitter` -> `new ng.core.EventEmitter`

File: modules/angular2/docs/cheatsheet/routing.md [hunk 3]
```diff
@@ -2,18 +2,26 @@
...
   { path: '/staticPath', component: ..., as: ...},
   { path: '/*wildCardParam', component: ..., as: ...}
 ])
 class MyComponent() {}`|`@RouteConfig`
+syntax(js):
+`ng.router.RouteConfig([
+  { path: '/:myParam', component: MyComponent, as: 'MyCmp' },
+  { path: '/staticPath', component: ..., as: ...},
+  { path: '/*wildCardParam', component: ..., as: ...}
+])
```
Comment (IgorMinar) [inline modules/angular2/docs/cheatsheet/routing.md]: this is not right - the fn below somehow needs to be chained with RouteConfig - @btford what's the right syntax?

File: modules/angular2/docs/cheatsheet/routing.md [hunk 4]
```diff
@@ -37,8 +45,10 @@ Creates a link to a different view based on a route instruction consisting of a
 
 
 @cheatsheetItem
-syntax(js ts):
+syntax(ts):
 `@CanActivate(() => { ... })class MyComponent() {}`|`@CanActivate`
+syntax(js):
+`ng.router.CanActivate(MyComponent, function() { ... })`|`ng.router.CanActivate`
```
Comment (IgorMinar) [inline modules/angular2/docs/cheatsheet/routing.md]: @btford please review. I think this one is correct.

**Comment by IgorMinar** [comment]
the rest looks good

**Pull request closed without merging**